app/grants/templates/grants/detail/info.html

```diff
@@ -0,0 +1,287 @@
+
+  <div>
+
+    <h1 class="font-title-xl my-4 font-weight-bold">
+      {% if is_admin %}
+        <textarea id="form--input__title" class="p-0 m-0 inline-edit" maxlength="150" disabled>{{ grant.title }}</textarea>
+      {% else %}
+        {{ grant.title }}
+      {% endif %}
+    </h1>
+
+     <!-- SUMMARY INFO -->
+    <div class="summary-info mb-4 pb-1">
+      <div class="font-body mb-2">
+        {% if is_admin %}
+          <textarea id="form--input__reference-url" class="p-0 m-0 inline-edit" disabled>{{ grant.reference_url }}</textarea>
+        {% else %}
+          <i style="width: 14px;" class="fab fa-github mr-2 text-center"></i>
+          <a href="{{ grant.reference_url }}" target="_blank" data-toggle="tooltip" data-html="true" data-placement="top" title="Project URL">
+            {{ grant.reference_url }}
+          </a>
+        {% endif %}
+      </div>
+
+      <div class="font-body mb-2">
+        <i style="width: 14px;" class="fab fa-ethereum mr-2 text-center"></i>
+        <a href="https://etherscan.io/address/{{ grant.admin_address }}" target="_blank" data-toggle="tooltip" data-placement="top" title="Recipient Funding Address">
+          {{ grant.admin_address }}
+        </a>
+      </div>
+
+      {% if grant.token_symbol %}
+        <div class="font-body mb-2">
+          <i style="width: 14px;" class="fas fa-coins mr-2 text-center"></i>
+          <p class="d-inline-block mb-0" data-toggle="tooltip" data-placement="top" title="ERC20 Token">
+            Grants accepts {{ grant.token_symbol }}
+          </p>
+        </div>
+      {% endif %}
+
+      {% if user_non_errored_subscription %}
+        <div class="font-body mb-2">
+          <i style="width: 14px;" class="fas fa-info-circle mr-2 text-center"></i>
+          <p class="mb-0 d-inline-block" data-toggle="tooltip" data-placement="top" title="Your Current Active Subscription">
+            You are contributing
+            {{ user_non_errored_subscription.amount_per_period|floatformat:2|intcomma }}
+            {{ user_non_errored_subscription.token_symbol }} every {{ user_non_errored_subscription.frequency }} {{ user_non_errored_subscription.frequency_unit }}
+          </p>
+        </div>
+      {% endif %}
+
+    </div>
+
+    {% if grant_is_inactive %}
+      <button type="button" class="button button--primary no-shadow font-weight-bold button--full py-3" disabled>
+        {% trans "Grant Has Ended" %}
+      </button>
+
+    {% elif user_non_errored_subscription and not is_admin %}
+
+      <a href="{% url 'grants:subscription_cancel' grant.id grant.slug user_subscription.id %}">
+        <button class="button button--primary button--warning button--full no-shadow font-weight-bold py-3">{% trans "Cancel Your Funding" %}</button>
+      </a>
+
+    {% else %}
+
+      <a href="{% url 'grants:fund' grant.id grant.slug %}">
+        <button class="btn btn-gc-blue button--full no-shadow font-weight-bold py-3">{% trans "Fund this Grant" %}</button>
```

Comment:
maybe we can just use .btn-lg class will increase the radius , font size, line-height and padding

Reply:
Hmm that would involve we updating the .btn-lg css cause out of the box it looks a lil meh 
Should I do that? 

##### original (matches design)

<img width="480" alt="Screenshot 2019-05-07 at 12 42 12 PM" src="https://user-images.githubusercontent.com/5358146/57279032-9dcf7d80-70c5-11e9-8f20-00149d1ceb7f.png">

##### using btn-lg
 . 
<img width="484" alt="Screenshot 2019-05-07 at 12 41 48 PM" src="https://user-images.githubusercontent.com/5358146/57279031-9d36e700-70c5-11e9-8f85-375bfcf31113.png">